letters/letters.py

```diff
@@ -3,50 +3,76 @@
 from redbot.core import commands
 
 # Define numbers -> emotes tuple
-nums = (':zero:', ':one:', ':two:', ':three:', ':four:', ':five:', ':six:', ':seven:', ':eight:', ':nine:')
+nums = (":zero:", ":one:", ":two:", ":three:", ":four:", ":five:", ":six:", ":seven:", ":eight:", ":nine:")
 
+# Define specials -> emotes dict
+specials = {"!": ":exclamation:", "?": ":question:", "#": ":hash:", "'": "'", ".": ".", ",": ","}
+
+allowed_chars = re.compile(r"[^a-z0-9!?\'.#, ]")
+
+def convert_char(char: str) -> str:
+    """Convert character to discord emoji"""
+    # Double space if char is space
+    if char == " ":
+        return "  "
+
+    # Convert to number emote if number
+    elif char.isdigit():
+        return f"{nums[int(char)]} "
+
+    # Convert to regional indicator emote if letter
+    elif char.isalpha():
+        return f":regional_indicator_{char}: "
+
+    # Convert to character emote
+    else:
+        return specials[char]
```

Comment:
This needs to be changed back to use string interpolation with a space at the end, matching lines 21 and 25.